### PR TITLE
demonstrate how VK email can be retrieved

### PR DESCRIPTION
**Pull request by coderofsalvation**
the VK api docs mention the 'email' scope, but there isn't any API call which returns an email field.
After lots of coffee and googling, it turns out the email only gets sent ONCE (with the token).

### CODE REVIEW

File: samples/VK.gs [hunk 1]
```diff
@@ -13,9 +13,9 @@ function run() {
   var service = getService();
   if (service.hasAccess()) {
     // GET requests require access_token parameter
-    var url = 'https://api.vk.com/method/groups.get?access_token=' +
-        service.getAccessToken();
+    var url = 'https://api.vk.com/method/users.get?version=5.89&fields=first_name,last_name,email&access_token=' + service.getAccessToken()
```
Comment (erickoledadevrel) [inline samples/VK.gs]: If email isn't being returned please remove it from the list of fields.

Comment (erickoledadevrel) [inline samples/VK.gs]: If it isn't required, please don't hard-code the version. Having the version in the URL makes this sample fall out of date faster.

File: samples/VK.gs [hunk 2]
```diff
@@ -13,9 +13,9 @@ function run() {
...
     var response = UrlFetchApp.fetch(url);
+    var email = service.getToken().email // email only retrievable from token
```
Comment (erickoledadevrel) [inline samples/VK.gs]: Please also log the email.

**Comment by erickoledadevrel** [comment]
Ping! Happy to merge with a few changes that I outlined.

File: samples/VK.gs (hunk 2 above)
Comment (coderofsalvation) [reply samples/VK.gs]: ok!👍

File: samples/VK.gs (hunk 1 above)
Comment (coderofsalvation) [reply samples/VK.gs]: ok!👍

Comment (coderofsalvation) [reply samples/VK.gs]: well, that is the funny part.
If you don't add email to the list of fields, it won't be exposed on the token (`service.getToken().email `)

**Comment by erickoledadevrel** [comment]
Thanks for the updates. Looks like you have a few style violations that you need to fix before the build is green: https://travis-ci.org/gsuitedevs/apps-script-oauth2/builds/525985438?utm_source=github_status&utm_medium=notification

**Comment by coderofsalvation** [comment]
thx. applied fixes to tame the beast. Almost there.

**Comment by erickoledadevrel** [comment]
Awesome, thanks again!

**Comment by coderofsalvation** [comment]
Nice, we tamed mr Travis! You're absolutely welcome.

**Pull request merged**